libs/reports-list/src/lib/reports-list/reports-list.component.ts

```diff
@@ -31,4 +40,11 @@ export class ReportsListComponent implements OnInit {
       console.log('The dialog was closed');
     });
   }
+
+  handleComingReport(eventData: VaccinesReport) {
```

Comment:
I don't like the name "Report", as it's ambiguous with the original name. Let's call it "Feedback".